exporters/metric/cortex/README.md

```diff
@@ -1,35 +1,121 @@
-# OpenTelemetry Go SDK Cortex Exporter
+# OpenTelemetry Go SDK Prometheus Remote Write Exporter for Cortex
 
-Work in progress exporter to send data from the OpenTelemetry Go SDK to Cortex using the Prometheus
-Remote Write API.
+This module contains an exporter that sends cumulative metrics data from the OpenTelemetry
+Go SDK to [Cortex](https://cortexmetrics.io/) using the Prometheus Remote Write API. While
+it is aimed at Cortex, it should work with other backends that ingest data with the same
+API.
 
-## Setting up an Exporter
+This exporter is push-based and integrates with the OpenTelemetry Go SDK's [push
+Controller](https://github.com/open-telemetry/opentelemetry-go/blob/master/sdk/metric/controller/push/push.go).
+The Controller periodically collects data and passes it to this exporter. The exporter
+then converts this data into
+[`TimeSeries`](https://prometheus.io/docs/concepts/data_model/), a format that Cortex
+accepts, and sends it to Cortex through HTTP POST requests. The request body is formatted
+according to the protocol defined by the Prometheus Remote Write API. See Prometheus's
+[remote storage integration
+documentation](https://prometheus.io/docs/prometheus/latest/storage/#remote-storage-integrations)
+for more details on the Remote Write API.
 
-Users can setup the Exporter with the `InstallNewPipeline` function. It requires a `Config` struct
-and returns a push Controller that will periodically collect and push data.
+See the `example` submodule for a working example of this exporter.
 
-Example:
+Table of Contents
+=================
+   * [OpenTelemetry Go SDK Prometheus Remote Write Exporter for Cortex](#opentelemetry-go-sdk-prometheus-remote-write-exporter-for-cortex)
+   * [Table of Contents](#table-of-contents)
+      * [Setting up the Exporter](#setting-up-the-exporter)
+      * [Configuring the Exporter](#configuring-the-exporter)
+      * [Securing the Exporter](#securing-the-exporter)
+         * [Authentication](#authentication)
+         * [TLS](#tls)
+      * [Instrument to Aggregation Mapping](#instrument-to-aggregation-mapping)
+      * [Error Handling](#error-handling)
+      * [Retry Logic](#retry-logic)
+
+## Setting up the Exporter
+
+Users only need to call the `InstallNewPipeline` function to setup the exporter. It
+requires a `Config` struct and returns a push Controller and error. If the error is nil,
+the setup is successful and the user can begin creating instruments. No other action is
+needed.
 
 ```go
+// Create a Config struct named `config`.
+
 pusher, err := cortex.InstallNewPipeline(config)
 if err != nil {
     return err
 }
 
-// Make instruments and record data
+// Make instruments and record data.
```

Comment:
Maybe indicate that the SDK was installed as the global SDK, with:
```
// Make instruments and record data using `global.MeterProvider`.
```

Reply:
Done!